front/src/common/Pathfinding/TypeAndPath.tsx

```diff
@@ -145,7 +145,7 @@ export default function TypeAndPath({ zoomToFeature }: PathfindingProps) {
 
   return (
     <div
-      className="type-and-path"
+      className="type-and-path w-100"
```

Comment:
Does it change something ?

Reply:
ok it does not change anything, i removed it